### ISSUE TITLE
Audio Input not transmitting

### ISSUE THREAD

**Issue by KevinMunson** [mumble-voip/mumble#5407]
### Description

When I try to talk to someone on a server, My voice does not transmit to the other person. 
In the settings, I run the audio wizard and I hear myself during that process. I am able to complete the entire wizard and hear myself at each step but still at the end the other person on the server can not hear me. 
 I also can use a different application and use the mic successfully. 

My mic is connected to an audio interface (ur 22mkII) and is a steinberg st-m01.  
The correct audio input device is chosen. 

I generally use push to talk, but it does not work for any of the mode(continuous, voice activity) 

### Steps to reproduce

1. I open mumble and connect to a server.
2. I try to talk.
3. They do not hear anything. 
4. Run audio wizard. 
5. Successfully complete this and hear myself in the process. 
6. Other person can still not hear me. 
7. Other person does see that I am trying to transmit

### Mumble version

1.3.4

### Mumble component

Client

### OS

Windows

### Reproducible?

Yes

### Additional information

I am using windows. I have used mumble for a long time with no issue. I was recently running 1.2.9 and ran into this issue and so I decided to get the latest version and got 1.3.3 and it still was having the same issue. So I got 1.3.4 and I am still getting this same issue. The server I am connecting to is 1.3.0 and was working before this and we are looking into updating that. It is currently on Debian 10 -> 11.  Not sure if this is relevant.

**Comment by Krzmbrzl** [comment]
@KevinMunson you closed the issue again. Does thst mean it is fixed now? If so, please share what you did to get it working again :)

**Comment by KevinMunson** [comment]
@Krzmbrzl  Yes I did solve the issue, but in an unexpected way: It wasnt my issue! So it turns out that I only had one interlocutor at the time, we apparently couldnt tell that they were the one actually not getting my output rather than my input being lost. When a different person joined the server to test, I was able to talk to them just fine. We will investigate why the other person was having the issue they were having.
